### ISSUE TITLE
Update the way stuff is added to the PyGuide and Auto-complete.

### ISSUE THREAD

**Issue by Moosichu** [pyland/pyland#281]
At the moment, in the config.jsonnet folder the structure of the PyGuide entries is as follows:
"pyguide_apis": {
		"number": 8,
		"0": "move_east(): This moves the present character towards the right by one step.",
		"1": "move_west(): This moves the present character towards the left by one step.",
		"2": "move_south(): This moves the present character towards the bottom by one step.",
		"3": "move_north(): This moves the present character towards the top by one step.",
		"4": "yell(): This has effect when it is called by a character that is facing a crocodile. This attracts the crocodile towards the yeller.",
		"5": "cut(): This cuts a vine that is present in front of a character",
		"6": "get_cuts_left(): This returns the number of cuts the person running the script has left",
		"7": "print(args): This prints the argument args to the terminal (args must be of type string)"
	}
, this is slightly confusing and it might be preferable to have it be as follows:

"pyguide_apis": {
		"move_east()" : "This moves the present character towards the right by one step.",
		 "move_west()": "This moves the present character towards the left by one step."
}

This would also make it easier to add things to the auto-complete as the auto-completer could read from the keys as well.

**Comment by asb** [comment]
Presumably it's written the way it is now in order to give control over ordering. In which case, an array is probably a better way to go (either an array of two-element arrays, or just bare strings).

**Comment by Moosichu** [comment]
Surely the order it is in in the jsonnet file should be the same order in which it is in the game? Or is order preservation not guaranteed with json?

**Comment by Moosichu** [comment]
Yeah, I think it should be moved out of the jsonnnet file anyway at some point, probably to the database as we will want to easily be able to provide translations to all the PyGuide entries anyway.

**Comment by asb** [comment]
A dictionary is an unordered key->value map. Some Json parsers do preserve the order when reading in, but not all. Jsonnet itself will re-order keys alphabetically for consistency in the generated .json.

**Comment by Moosichu** [comment]
Ok, so maybe moving straight to the database anyway would be better. Or are there reasons why this might not be a good idea?

**Comment by sams25** [comment]
Should we move them to the database then?
